src/app/systelab-components/contextmenu/context-menu-item.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="hasIcons && isIconEnabled"
+     class="slab-icon-container rounded d-flex text-center justify-content-center align-items-center"
+     [style.background-color]="action.backgroundIconColor">
+    <i class="slab-flex-1 slab-menu-icon {{action.iconClass}}" [style.color]="action.iconColor"
+       [ngClass]="{'text-white': !action.iconColor}"></i>
+</div>
+<a class="slab-flex-1" [style.padding-left]="hasIcons && !isIconEnabled?'20px':'0'" [ngClass]="{'ml-2': hasIcons}">{{action.actionText}}</a>
```

Comment:
Avoid fixed sizes. The size must be increased based upon $slab-size-percentage. Redefine the 20px.

Reply:
Pending to discuss with @VictorDeBlas